src/waveletTypes.jl

```diff
@@ -18,6 +18,20 @@ class(::Morlet) = "Morlet"; name(::Morlet) = "morl"; vanishingmoments(::Morlet)
 const morl = Morlet()
 Base.show(io::IO, x::Morlet) = print(io, "Morlet mean $(x.σ)")
 
+
+struct Morse <: ContinuousWavelets.ContWaveClass
+    ga::Float64
+    be::Float64
+    cf::Float64
+end
+
+N = 100; ga = 3; be = 1; cf = 1;
```

Comment:
I'm guessing this is just leftovers from experiments. just a reminder to remove it.